Jenkinsfile

```diff
@@ -18,39 +17,51 @@ def localSrcFolder = '.'
 def lcovFile = './test-output/lcov.info'
 def timeoutInMinutes = 5
 
-
 node {
   checkout scm
   try {
+    stage('Set GitHub status as pending'){
+      defraUtils.setGithubStatusPending()
+    }
     stage('Set branch, PR, and containerTag variables') {
       (pr, containerTag, mergedPrNo) = defraUtils.getVariables(repoName)
```

Comment:
Need to reference the CSProj version number here using the shared library function